instantsearch.js/render-state/src/app.js

```diff
@@ -32,11 +32,8 @@ const emptyHits = connectHits(({ hits, widgetParams }) => {
     return;
   }
 
-  const refinementListsUiState = search.getUiState()[indexName].refinementList;
-  const isPearRefined =
-    refinementListsUiState &&
-    refinementListsUiState.brand &&
-    refinementListsUiState.brand.includes('Pear');
+  const brandUiState = search.getUiState()[indexName].refinementList?.brand;
```

Comment:
still using ui state, no?

Reply:
🤦‍♂️ thanks, i accidentally reapplied part of my changes.